float64[] palmmat4
float64[] currentangles
---
float64[] resultangles
string success